components/patient-access-rules/ui/src/main/resources/PhenoTips/AccessRightsManagement.xml

```diff
@@ -1055,14 +1055,14 @@
           collaboratorText = "$escapetool.javascript($services.localization.render('phenotips.accessRightsManagement.collaboratorsSummary.noCollaborators', ['__beginHtml__', '__endHtml__']))".replace("__beginHtml__", beginHtml).replace("__endHtml__", endHtml);
         } else {
           var collaborator = data.collaborators.collaborators[0];
-          collaborator.name = collaborator.name || XWiki.Model.resolve(collaborator.id, XWiki.EntityType.DOCUMENT).name;
+          var safeCollaboratorName = (collaborator.name || XWiki.Model.resolve(collaborator.id, XWiki.EntityType.DOCUMENT).name).escapeHTML();
```

Comment:
Since the variable `collaborator` was only used for injection, it might be permissible to leave `collaborator.name` as possibly null and use `safeCollaboratorName` for injection.